### PR TITLE
Completed Loops

### PR DESCRIPTION
**Pull request by Alex-ArceJr**
Screen Shot 2023-03-04 at 2 46 54 PM

### CODE REVIEW

File: prep-javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+function forLoop1() {
+  const array = [];
+  for (let i = 0; i < 10; i++) {
+    array.push(i);
+  }
+  return array;
+}
+console.log('forLoop1 output', forLoop1());
+
+function forLoop2() {
```
Comment (BrettAlbright) [inline prep-javascript-loops/main.js:32]: for this function, you don't want to return an array, although you did that well! You just want to console log the string each time the loop decrements with the new index value.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
great work! one small update!

**Comment by Alex-ArceJr** [comment]
Screen Shot 2023-03-04 at 2 58 05 PM

**Comment by Alex-ArceJr** [comment]
Screen Shot 2023-03-04 at 2 59 11 PM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**